personality/wn/machine-features.pan

```diff
@@ -9,6 +9,26 @@ variable MACHINE_FEATURES_LIST ?= list(
     'total_cpu',
 );
 
+variable MACHINE_FEATURES_SCRIPT ?= 'mjf';
```

Comment:
Does this really need to be customisable? The following code would be much simpler if it wasn't.

Reply:
Yes, it should be customizable. The script can be modified to fit the configuration of the site and the batch scheduler.

Reply:
Ok, thanks.